src/components/Book.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ book }) => {
+const Book = ({ book, onClick }) => {
```

Comment:
we want to name our props properly, onClick is the event listener for click action in React so ideally, we do not want to have other functions with the same name, this is basically for readability reasons

Reply:
Yes, we will fix the naming convention issue. Thank you for your feedback! 👍